src/helpers/WLClasses.hpp

```diff
@@ -133,9 +133,10 @@ struct SKeyboard {
     std::string        xkbFilePath = "";
 
     SStringRuleNames   currentRules;
-    int                repeatRate  = 0;
-    int                repeatDelay = 0;
-    int                numlockOn   = -1;
+    int                repeatRate        = 0;
+    int                repeatDelay       = 0;
+    int                numlockOn         = -1;
+    bool               resolveBindsBySim = false;
```

Comment:
typo?

Reply:
Yup, that one got past me. Fixed it. Thanks for the review!